include/gazebo_video_stream_widget.h

```diff
@@ -19,10 +19,7 @@
 
 #include <gazebo/common/Plugin.hh>
 #include <gazebo/gui/GuiPlugin.hh>
-#ifndef Q_MOC_RUN  // See: https://bugreports.qt-project.org/browse/QTBUG-22829
-# include <gazebo/transport/transport.hh>
-# include <gazebo/gui/gui.hh>
-#endif
+#include <gazebo/transport/transport.hh>
```

Comment:
Why is this removed? There must be a a purpose for this. If it is related with the Gazebo version, then you need to wrap this up in preprocessor directives that make the usage of one or another dependent of the Gazebo version.